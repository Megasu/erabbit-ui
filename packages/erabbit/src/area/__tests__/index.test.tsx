import { Area } from '..'
import { mount } from '@vue/test-utils'

test('should render default Area', () => {
  const wrapper = mount(Area, {
    slots: {
      default: 'Submit'
    }
  })
  expect(wrapper.html()).toMatchSnapshot()
})
