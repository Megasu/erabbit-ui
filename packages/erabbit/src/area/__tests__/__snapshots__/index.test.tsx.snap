// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render default Area 1`] = `
"<div class=\\"er-area\\">
  <div class=\\"er-area__input\\">
    <!----><span>请选择地区</span><span class=\\"er-area__arrow\\"> &gt;</span>
  </div>
  <!---->
</div>"
`;
